packages/compartment-mapper/src/types/node-powers.d.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * @file These interfaces describe the powers needed in `node-powers.js` to
+ * adapt host capabilities for the compartment mapper.
+ */
+
+/* eslint-disable no-use-before-define */
+
+/** For creating {@link ReadPowers} */
```

Comment:
```suggestion
/** For creating `ReadPowers` */
```

The `@link` is now broken. There may be a few broken `@link`s in these files

Reply:
Did a holistic pass changing `@link` to bare words or ticked words if the referent was in a different file. Push forthcoming.